net.purrdata.PurrData.yml

```diff
@@ -0,0 +1,118 @@
+id: net.purrdata.PurrData
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
+command: pd-wrapper.sh
+finish-args: [
+    # Display
+    '--share=ipc', '--socket=x11', '--socket=wayland', '--device=dri',
+    # Audio access
+    '--socket=pulseaudio',
+    # Filesystem access (pd doesn't yet work with portals)
+    '--filesystem=host',
+    '--filesystem=home',
+    # MIDI access (Flatpak doesn't yet support exposing only MIDI devices)
+    '--device=all'
+]
+modules:
+- name: purr-data
+  sources:
```

Comment:
While it's not written down anywhere, `sources` as the last element of module is preferred as it's easier to review.